Makefile

```diff
@@ -59,3 +59,5 @@ run_python_builder:
 		-v $(SELDON_CORE_LOCAL_DIR):/work \
 		seldonio/python-builder:0.2 bash
 
+protos_go:
+	make -C ./go compile_proto
```

Comment:
We tend to keep the root `Makefile` clean, so I'd remove this rule and just call `make -C go compile_proto` directly on the CI/CD pipeline.

Reply:
Sounds fair, have reverted root Makefile